docker-compose.yml

```diff
@@ -655,13 +548,11 @@ services:
       DEPLOYMENT_NAME: "${DEPLOYMENT_NAME}"
       GRAPL_TEST_USER_NAME: "${DEPLOYMENT_NAME}-grapl-test-user"
     tty: true
-    depends_on:
-      dgraph:
-        condition: service_started
-      localstack:
-        condition: service_healthy
-      pulumi:
-        condition: service_completed_successfully
+    #    depends_on:
+    #      dgraph:
+    #        condition: service_started
+    #      pulumi:
+    #        condition: service_completed_successfully
```

Comment:
Out of curiosity, does the `service_completed_successfully` work if the `pulumi` was run by a separate command?

Reply:
I'm a bit suspicious of this as well.

Reply:
Since we're moving Provisioner to Nomad, this doesn't matter and I'm ripping it out soon

Reply:
I've backed this out. Originally I was trying to run nomad-local-infra, run pulumi, run the provisioner and then run nomad-grapl-core. Since we're now running provisioner via nomad, this can be uncommented